hw/dv/tools/vcs/common_cov_excl.cfg

```diff
@@ -0,0 +1,15 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// Based on our Comportable IP spec, these TL pins are reserved / unused and hence, tied off.
+-node tb.dut *tl_i.a_user.rsvd
+-node tb.dut *tl_i.a_param
+-node tb.dut *tl_o.d_param
+-node tb.dut *tl_o.d_opcode[1]
+-node tb.dut *tl_o.d_opcode[2]
+-node tb.dut *tl_o.d_sink
+-node tb.dut *tl_o.d_user.rsp_intg[6]
```

Comment:
@tjaychen 

rsp_intg[6] will be never changed as only the LSB 4 bits in tl_d2h_rsp_intg_t can be changed, and we mask these LSB 4 bits to 0 due to `data_o[63] = ^(data_o & 64'h01FBDDA769A46910);`. So we have to exclude toggle coverage for this bit. LMK if this isn't right.
```
typedef struct packed {
    tl_d_op_e                     opcode;
    logic  [top_pkg::TL_SZW-1:0]  size;
    // Temporarily removed because source changes throughout the fabric
    // and thus cannot be used for end-to-end checking.
    // A different PR will propose a work-around (a hoaky one) to see if
    // it gets the job done.
    //logic  [top_pkg::TL_AIW-1:0]  source;
    logic                         error;
  } tl_d2h_rsp_intg_t;

```
```
module prim_secded_64_57_enc (
  input        [56:0] data_i,
  output logic [63:0] data_o
);

  always_comb begin : p_encode
    data_o = 64'(data_i);
    data_o[57] = ^(data_o & 64'h0103FFF800007FFF);
    data_o[58] = ^(data_o & 64'h017C1FF801FF801F);
    data_o[59] = ^(data_o & 64'h01BDE1F87E0781E1);
    data_o[60] = ^(data_o & 64'h01DEEE3B8E388E22);
    data_o[61] = ^(data_o & 64'h01EF76CDB2C93244);
    data_o[62] = ^(data_o & 64'h01F7BB56D5525488);
    data_o[63] = ^(data_o & 64'h01FBDDA769A46910);
  end
```

Regarding the `source`, do we have a better idea to apply integrity for it now? only the MSB 2 bits of source may be changed by xbar as we have 3 hosts, the other 6 bits won't be changed. can we consider to have integrity check for those LSB 6 bits?

Reply:
IMO the cfg file deserves a comment explaining this on the line excluding `rsp_intg[6]`.

Reply:
Added comment.

Reply:
@weicaiyang ah yes this is fine, just a result of how ECC behaves. This does mean however, if we add more information to the integrity in the future, OR if the ECC bit selection changes, this masking will have to be different as well...

what do you think would be the best way to handle that? This could be a bit dynamic depending on the design.

Reply:
I don't think we can automatically handle this. When design is updated, we need to manually change this exclusion.
But if we have a clear idea to add source to encode the ECC, then we can update design and we don't need this now.

Reply:
Shouldn't UNR detect this?

Reply:
@sriyerg yes, UNR can also report this line. I also tagged it as UNR

Reply:
Sorry, what I mean is we shouldn't have to add this. It should come from the UNR exclusion file.

Reply:
since these are common UNR, I think we should add them here so that we don't need to review them in each block. WDYT?
Please see the bullet 3 in this link. 
https://github.com/lowRISC/opentitan/blob/master/doc/ug/dv_methodology/index.md#unreachable-coverage-analysis

Reply:
IMO it is better if this comes as a part of UNR flow, rather than having it excluded at compile time, because it was inferred from the internal knowledge of the design, which as @tjaychen suggests above, could change in future. Each block DV owner _needs_ to run UNR so this should show up there instead anyway.

Reply:
In this case, we also need to remove below in this file, as they're also UNR.
```
-node tb.dut *tl_o.d_param
-node tb.dut *tl_o.d_opcode[1]
-node tb.dut *tl_o.d_opcode[2]
-node tb.dut *tl_o.d_sink
```

yes, future design may change. If it's in each block, we need to change in every block rather than just one place. Let's discuss this more offline.